test/unit/mdc-top-app-bar/foundation.test.js

```diff
@@ -58,3 +77,68 @@ test('click handler removed from navigation icon during destroy', () => {
   foundation.destroy();
   td.verify(mockAdapter.deregisterNavigationIconInteractionHandler('click', td.matchers.isA(Function)));
 });
+
+test('short top app bar: on scroll listener is registered', () => {
```

Comment:
Nit: re-word to "short top app bar: scroll listener is registered on init"

(This was hard for me to read at first, I was reading it as "on scroll, listener is registered")